packages/contractkit/src/wrappers/TransactionBuilder.ts

```diff
@@ -0,0 +1,103 @@
+import { keccak256 } from 'ethereumjs-util'
+import Contract from 'web3/eth/contract'
+
+import { Address, CeloContract } from '../base'
+import { ContractKit } from '../kit'
+import { CeloTransactionObject, NumberLike, toBuffer } from './BaseWrapper'
+import { encodedTransactions, Transaction } from './Governance'
+
+export interface JSONTransaction {
+  value: NumberLike
+  celoContractName: CeloContract
+  methodName: string
+  args: any[]
+}
+
+type Web3Method = Contract['methods'][keyof Contract['methods']]
+export class TransactionBuilder {
```

Comment:
I would call this ProposalBuilder even it's just txs. They are also partial tx for Proposal only